### ISSUE TITLE
Cannot reproduce the ideal accuracy of NVAN

### ISSUE THREAD

**Issue by Kewenjing1020** [jackie840129/STE-NVAN#12]
Hi,
I reproduce the training procedure and get the results:
> baseline: R1=88.54%(+1.12%), mAP=80.71% (2 GPUs)
> NVAN: R1=89.55%(-0.45%), mAP=83.05% (3GPUs)
> NVAN: R1=89.60%(-0.40%), mAP=82.31% (2GPUs)
> STE-NVAN: R1=88.08%(-0.61%), mAO=79.83%(2GPUs)

The training setting and pytorch version is exactly the same as given, the only difference might be that i use several GPUs to train. 
The baseline is 1.12% higher than you reported, but NVAN and STE-NVAN is lower. Do you have any ideas on increase NVAN accuracy?

**Comment by jackie840129** [comment]
For NVAN, I think the number of GPU may influence the acc (owing to the separate batch norm on each GPU).
For STE-NVAN, you can use the weight after NVAN as initialization and use a smaller lr for fine-tuning.
